src/mainwindow.cpp

```diff
@@ -2687,6 +2688,7 @@ void MainWindow::onProducerOpened(bool withReopen)
         Util::getHash(*MLT.producer());
         ui->actionPaste->setEnabled(true);
     }
+    ui->actionSave->setEnabled(true);
```

Comment:
Maybe we should change the save action's enabled state based on when the `QMainWindow::setWindowModified()` instead of this one case. There are 12 lines where that is set in `mainwindow.cpp`. However, the window-modified state is not super critical at the moment. It provides a window decoration to indicate and affects `continueModified()`. I would hate to put the tool into a situation where it will not save. What do you think?

Reply:
> I would hate to put the tool into a situation where it will not save

This is my greatest fear and the reason I put that change in a separate commit. Also, the "action enable" change is not material for the fix (as you correctly observe below). I notice that some applications do not allow saving an empty file/project while others do. So I am not sure how important it is to disable the action. If you like the idea, I am willing to put more time into making a better implementation that is more comprehensive. But maybe wait until there will be a beta release.